relay-server/src/actors/store.rs

```diff
@@ -50,6 +51,49 @@ pub enum StoreError {
     NoEventId,
 }
 
+/// Internal wrapper of a message sent through an `StoreAddr` with return channel.
```

Comment:
Any chance you could add a TODO here indicating that we need to unify it with the Message from the healthcheck actor?